tests/bears/DebugBearsTest.py

```diff
@@ -1,9 +1,22 @@
+import multiprocessing
 import unittest
 import sys
 
 from io import StringIO
 
-from coalib.bears.Bear import Debugger, debug_run
+from coalib.bears.Bear import Bear, Debugger, debug_run
+from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpf3rh_38b/tests/bears/DebugBearsTest.py
+++ b/tmp/tmpf3rh_38b/tests/bears/DebugBearsTest.py
@@ -86,7 +86,8 @@
         # result, output = execute_debugger(["settings x=3; y= 'abc2'; z=abc;"
         #                                    'some_key=2', 'q', 'settings', 'c',
         #                                    'q'], my_bear.run, *args, **kwargs)
-        input = StringIO("settings x=3;y= 'abc2';z=abc;some_key=2\nq\nsettings\nc\nq")
+        input = StringIO(
+            "settings x=3;y= 'abc2';z=abc;some_key=2\nq\nsettings\nc\nq")
         output = StringIO()
         dbg = Debugger(stdin=input, stdout=output)
         debug_run(my_bear.run, dbg, *args, **kwargs)
```